control/control.go

```diff
@@ -90,8 +90,20 @@ type pluginControl struct {
 	grpcServer  *grpc.Server
 	closingChan chan bool
 	wg          sync.WaitGroup
+
+	subscriptionGroups ManagesSubscriptionGroups
+}
+
+type corePlugin struct {
```

Comment:
It implements core.Plugin interface, I think naming is OK in this case. What would you suggest instead?